.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -42,7 +44,7 @@ Debug output:
 ```
 </details>
 
-Paste the logs from the lsp-client, e.g. for [VS Code](https://github.com/haskell/vscode-haskell#troubleshooting)
+Paste the logs from the lsp-client, you can check instructions about for VS Code [here](https://github.com/haskell/vscode-haskell#troubleshooting)
```

Comment:
Again, I think we should refer people to the relevant section of the troubleshooting page.

Reply:
hmm https://github.com/haskell/vscode-haskell#troubleshooting would not be the relevant page?